host/sgx/xstate.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef _OE_XSTATE_H
+#define _OE_XSTATE_H
+
+#include <openenclave/internal/sgxtypes.h>
```

Comment:
Is this inclusion necessary? No function signature references a type from sgxtypes.h here.

Reply:
Yes, this inclusion is necessary as SGX_XFRM_AVX and SGX_XFRM_LEGACY are defined here.

Reply:
You probably want to comment that this is a convenience inclusion then to keep a different maintainer from removing this inclusion in the future. (i.e. it makes the two definitions available to users of `xstate.h` by default)

Reply:
Sorry, my previous remark was incorrect. I do not need to include sgxtypes.h. Only types.h is needed for uint64_t. Made the change.